### PR TITLE
Add compatibility with Symfony 3.4

### PR DESCRIPTION
**Pull request by Wharenn**
## Changelog
```markdown
### Fixed
- Fixed compatibility with Symfony 3.4
```

## Subject
This PR add a workaround for twig form theming in Symfony 3.4 based on similar implementations in other Sonata bundles.

### CODE REVIEW

File: src/Controller/FragmentAdminController.php [hunk 1]
```diff
@@ -66,4 +72,29 @@ public function formAction()
 
         return $response;
     }
+
+    /**
+     * Sets the admin form theme to form view. Used for compatibility between Symfony versions.
+     *
+     * @param FormView $formView
```
Comment (greg0ire) [inline src/Controller/FragmentAdminController.php]: Please remove this phpdoc

File: src/Controller/FragmentAdminController.php [hunk 2]
```diff
@@ -66,4 +72,29 @@ public function formAction()
...
+     * @param mixed    $theme
+     */
+    private function setFormTheme(FormView $formView, $theme): void
+    {
+        $twig = $this->get('twig');
+        // BC for Symfony < 3.2 where this runtime does not exists
```
Comment (greg0ire) [inline src/Controller/FragmentAdminController.php]: "does not exist"

**Comment by Wharenn** [comment]
Thanks @greg0ire, the PR is updated :)

**Review by kunicmarko20 — DISMISSED** [review]

**Comment by Wharenn** [comment]
There was a typo, I was calling `getFilterTheme` since we were not in `listAction`. Replaced it with `getFormTheme`.

**Review by greg0ire — APPROVED** [review]

**Review by core23 — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @Wharenn !

**Pull request merged**